experimental/dds/tree2/src/simple-tree/proxies.ts

```diff
@@ -871,37 +914,70 @@ interface ExtractedFactoryContent<T extends InsertableTypedNode<TreeNodeSchema>>
  * }
  * ```
  */
+export function extractFactoryContent<
+	T extends InsertableTypedNode<FieldNodeSchema> | Unhydrated<FieldNodeSchema>,
+>(input: T, insertedAtIndex: number): ExtractedFactoryContent<T>;
```

Comment:
Probably worth updating the function comment with an `@param` comment for `insertedAtIndex`

Reply:
Sure!